eng/versioning/version_client.txt

```diff
@@ -57,7 +57,7 @@ com.azure:azure-aot-graalvm-support;1.0.0-beta.2;1.0.0-beta.3
 com.azure:azure-aot-graalvm-support-netty;1.0.0-beta.2;1.0.0-beta.3
 com.azure:azure-aot-graalvm-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-aot-graalvm-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-communication-chat;1.2.4;1.3.0-beta.1
+com.azure:azure-communication-chat;1.2.5;1.2.5
```

Comment:
We only need to update the "current" version (right). The " dependency" version (left) refers to the latest released version of the library.
```suggestion
com.azure:azure-communication-chat;1.2.4;1.2.5
```